tools/bwa.wdl

```diff
@@ -30,23 +30,29 @@ task bwa_aln {
             n_cores=$(nproc)
         fi
 
-        mkdir bwa
+        mkdir /tmp/bwa
 
-        tar -C bwa -xzf ~{bwadb_tar_gz}
-        PREFIX=$(basename bwa/*.ann ".ann")
+        tar -C /tmp/bwa -xzf ~{bwadb_tar_gz}
```

Comment:
This change would be inconsistent with the star task. I'm also not sure this is a good approach. There is no guarantee that `/tmp` in a container is mounted to a location with sufficient storage. In our Singularity config, we set `--workdir` to the task directory. So there `/tmp` would be a subdirectory of the main task directory.

Reply:
I could've sworn I read somewhere a long time ago that it was a WDL best practice to put large files in `/tmp`, but I just did some googling and can't find any reference to that practice. Don't remember the rationale for that statement, so can no longer justify this change.

I think some other tasks with large references I've written I've put them in `/tmp`, but if `/tmp` might not have sufficient storage based on backend, I think I should go through and change all those to use CWD.